File: AppGenerateFiles/dbo/Model/Detail_Movimiento.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Detail_Movimiento : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? id_detalle { get; set; }
       public int? id_movimiento { get; set; }
       public int? id_cuenta { get; set; }
       public Double? debito { get; set; }
       public Double? credito { get; set; }
       public Double? tasa_cambio { get; set; }
       public DateTime? fecha { get; set; }
       public Double? debito_dolares { get; set; }
       public Double? credito_dolares { get; set; }
       public Double? monto_inicial { get; set; }
       public Double? monto_final { get; set; }
       public Double? monto_inicial_dolares { get; set; }
       public Double? monto_final_dolares { get; set; }
       public string? moneda { get; set; }
       public Double? tasa_cambio_compra { get; set; }
       [ManyToOne(TableName = "Catalogo_Cuentas", KeyColumn = "id_cuentas", ForeignKeyColumn = "id_cuenta")]
       public Catalogo_Cuentas? Catalogo_Cuentas { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Detail_Movimiento.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Catalogo_Cuentas }  from './Catalogo_Cuentas.js'
class Detail_Movimiento extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id_detalle;
   /**@type {Number}*/ id_movimiento;
   /**@type {Number}*/ debito;
   /**@type {Number}*/ credito;
   /**@type {Number}*/ tasa_cambio;
   /**@type {Date}*/ fecha;
   /**@type {Number}*/ debito_dolares;
   /**@type {Number}*/ credito_dolares;
   /**@type {Number}*/ monto_inicial;
   /**@type {Number}*/ monto_final;
   /**@type {Number}*/ monto_inicial_dolares;
   /**@type {Number}*/ monto_final_dolares;
   /**@type {String}*/ moneda;
   /**@type {Number}*/ tasa_cambio_compra;
   /**@type {Catalogo_Cuentas} ManyToOne*/ Catalogo_Cuentas;
}
export { Detail_Movimiento }
